integration/test/crmapping/crmapping_test.go

```diff
@@ -354,21 +350,19 @@ func verifyCR(t *testing.T, client client.Client, testName string, obj runtime.O
 		t.Fatalf("err = %#q, want %#v", microerror.JSON(err), nil)
 	}
 
-	//
-	// XXX: Workaround to exclude timestamp differences from comparison.
-	//		Currently e.g. AzureConfig.Status.Cluster has few fields with
-	//		lastTransitionTime that contains timestamp. When executing e.g.
-	//		test that creates AzureConfig from CAPI + CAPZ types, it injects
-	//		time.Now() in those fields and this would always differ from golden
-	//		files that represent one point in time.
-	//
-	re := regexp.MustCompile(`lastTransitionTime: "\d+-\d+-\d+T\d+:\d+:\d+Z"`)
-	timeNow := time.Now()
-	goldenFile = re.ReplaceAll(goldenFile, []byte(fmt.Sprintf(`lastTransitionTime: "%s"`, timeNow)))
-	out = re.ReplaceAll(out, []byte(fmt.Sprintf(`lastTransitionTime: "%s"`, timeNow)))
-
-	// Final comparison of golden version vs. one generated by test.
-	if !bytes.Equal(out, goldenFile) {
-		t.Fatalf("\n\n%s\n", cmp.Diff(string(goldenFile), string(out)))
+	objMap := make(map[string]interface{})
+	err = yaml.Unmarshal(out, objMap)
+	if err != nil {
+		t.Fatalf("err = %#q, want %#v", microerror.JSON(err), nil)
+	}
+
+	goldenMap := make(map[string]interface{})
+	err = yaml.Unmarshal(goldenFile, goldenMap)
+	if err != nil {
+		t.Fatalf("err = %#q, want %#v", microerror.JSON(err), nil)
+	}
+
+	if !cmp.Equal(objMap, goldenMap) {
```

Comment:
Had to change the comparison because map ordering is not guaranteed in Go so labels were breaking in tests.